recipes/xcfun/conda_build_config.yaml

```diff
@@ -0,0 +1,17 @@
+python:
+    - 3.7
+    - 3.6
+    - 2.7.16
+
+PYMOD_INSTALL_LIBDIR:
+    - '/python3.7/site-packages'
+    - '/python3.6/site-packages'
+    - '/python2.7/site-packages'
```

Comment:
c-f conventions won't like this cbc.yaml. You can get rid of the pymod_install_libdir by following my pattern in https://github.com/conda-forge/staged-recipes/pull/8804

Reply:
Thanks Lori! I'll give it a try. I was trying to understand how to test this locally, rather than burn precious CI minutes.